mkposters/mkposter.py

```diff
@@ -21,25 +21,30 @@ def md_to_html(md: str):
     )
 
 
+def excludeHiddenDirs(path, subdirs):
+    return [p for p in subdirs if p.startswith(".")]
+
 def parse(datadir: pathlib.Path, tempdir: pathlib.Path, join_scss_file: pathlib.Path):
     if (datadir / "icons").exists():
         raise ValueError
     if (datadir / "stylesheets").exists():
         raise ValueError
-    shutil.copytree(datadir, tempdir, dirs_exist_ok=True)
+    shutil.copytree(datadir, tempdir, dirs_exist_ok=True, ignore=excludeHiddenDirs)
 
     md_file = tempdir / "poster.md"
     html_file = tempdir / "index.html"
     css_file = tempdir / "style.css"
 
     with md_file.open() as f:
         contents = f.read()
-    banner, left_body, right_body = contents.split("--split--")
-
-    banner = md_to_html(banner)
-    left_body = md_to_html(left_body)
-    right_body = md_to_html(right_body)
-    html_out = rf"""<!doctype html>
+    bodies = contents.split("--split--")
+
+    bodies = [md_to_html(b) for b in bodies]
+    banner = bodies[0]
+    left_body = bodies[1]# Still capture left and right seperatly for backwards compadibility
```

Comment:
Take a look at [CONTRIBUTING.md](https://github.com/patrick-kidger/mkposters/blob/main/CONTRIBUTING.md) regarding pre-commit hooks.

(The nit that would catch here is spaces before comments.)

Also, typo in "compadibility"